janus_core/helpers/correlator.py

```diff
@@ -196,6 +198,7 @@ class Correlation:
 
     def __init__(
         self,
+        n_atoms: int,
```

Comment:
Given that the order of these isn't particularly intuitive, might it make sense to start with `*,`, to force passing them as key words?